hipo_meal/settings.py

```diff
@@ -144,5 +144,6 @@
     ),
     'DEFAULT_PERMISSION_CLASSES': (
         'rest_framework.permissions.IsAuthenticatedOrReadOnly',
-    )
+    ),
+    'DATETIME_FORMAT': "%Y-%m-%d %H:%M",
```

Comment:
Formating the date is work of the clients. By default `iso-8601` is used by drf. It returns timezone aware date time object. This allow clients to format it in any way they want.